xrdp/xrdp.ini.in

```diff
@@ -53,6 +53,8 @@ use_fastpath=both
 #require_credentials=true
 ; You can set the PAM error text in a gateway setup (MAX 256 chars)
 #pamerrortxt=change your password according to policy at http://url
+; Cache Glyph Revison 2. default: false
+use_cache_glyph_v2=true
```

Comment:
I'm sure about the necessity of this switch. If this is set to false, no fonts displayed to mac client. I think always-on is OK on the server side. Of course, if the client doesn't support v2, the server should not do that.